src/ucp/api/ucp.h

```diff
@@ -3312,12 +3312,12 @@ ucp_atomic_fetch_nb(ucp_ep_h ep, ucp_atomic_fetch_op_t opcode,
  * @param [in] ep          UCP endpoint.
  * @param [in] opcode      One of @ref ucp_atomic_fetch_op_t.
  * @param [in] buffer      Address of operand for atomic operation. For 
- *                         @ref CSWAP operation, this is the value the remote
- *                         memory buffer is compared with. For @ref SWAP
+ *                         CSWAP operation, this is the value the remote
```

Comment:
add ref to the enum value

Reply:
do you mean replace CSWAP by UCP_ATOMIC_FETCH_OP_CSWAP?

Reply:
yes